src/lib/services/types/block.ts

```diff
@@ -21,18 +21,23 @@ import { zTx } from "./tx";
 const zNullableValidator = z.nullable(
   z
     .object({
-      validator_address: zValidatorAddr.nullable(),
+      validator_address: zValidatorAddr.nullish(),
+      operator_address: zValidatorAddr.nullish(),
```

Comment:
isn't this the same as validator_address? do we really need to have 2 fields for the same data?